source/api_reference/payment_event_reference/index.html.md.erb

```diff
@@ -112,6 +112,7 @@ After you send your request to get a payment’s events, you’ll receive a resp
 | `updated`        | date (ISO 8601) | When this payment's `state` changed.<br><br>This value uses Coordinated Universal Time (UTC) and ISO-8601 format - `YYYY-MM-DDThh:mm:ss.sssZ`.                                                                                                                                                                                            |
 | `state`          | object          | Contains information about what the `status` was updated to and whether this payment's journey finished.                                                                                                                                                                                                                                |
 | `state.status`   | string          | What the payment's status was updated to.<br><br>Possible values are:<br><br><li>`created`<li>`started`<li>`submitted`<li>`capturable`<li>`success`<li>`failed`<li>`cancelled`<li>`error`</li><br>You can [read more about the payment status lifecycle](/api_reference/#payment-status-lifecycle). |
+| <nobr>`state.can_retry`| boolean         | Indicates if you can try to take another payment using the associated agreement for recurring payments.<br><br>If `can_retry` is `true`, you can use this agreement to try to take another recurring payment.<br><br>If `can_retry` is `false`, you cannot take another recurring payment with this agreement.<br><br>You can [read more about handling failed recurring payments](/recurring_payments/#handling-failed-recurring-payments).                                                                                                                                                                                        |
```

Comment:
I think it's worth adding that it's not always present. It's only present if:
- the payment was through RCP
- and the payment was declined

Reply:
Really good point. I'll add that in.